sopel_modules/rat_board.py

```diff
@@ -2037,3 +2037,21 @@ def cmd_invalid(bot, trigger, caseid):
 
     except:
         bot.reply('Couldn\'t find a case with id ' + str(caseid) + ' or other APIError')
+
+
+@commands('lang', 'language')
+@parameterize('rw', usage='<case number or name> <language code>')
+@require_permission(Permissions.Rat)
+def cmd_lang(bot, trigger, case, lang):
+    """
+    Sets a case's language
+    """
+    lang = lang.lower()
+    if lang in ["en", "cs", "nl", "fr", "de", "no", "pt", "ru", "es", "sv"]:
+        with bot.memory['ratbot']['board'].change(case):
+            case.data.update({'langID': lang})
+
+        save_case_later(bot, case, forceFull=True)
+        bot.say('Language on case %s changed to %s.' % (case.client_name, lang))
```

Comment:
please use .format rather than `%` substitution, as it is depricated.
As noted in [python v3 documentation](https://docs.python.org/release/3.1.5/library/stdtypes.html#old-string-formatting-operations):
>Note The formatting operations described here are obsolete and may go away in future versions of Python. Use the new String Formatting in new code.

Reply:
I've been spoiled by too much C I suppose